### DISCUSSION TITLE
Realm migration issue (One or more errors occurred. (SQLite Error 1: 'no such column: s.InstantiationInfo'.))

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#16700] [Q&A] [answered]
Please drag the "attach_me.zip" file here
[attach_me.zip](https://github.com/ppy/osu/files/7963671/attach_me.zip)
!

**Comment by bdach** [comment]
just for confirmation purposes, did you not upgrade lazer for a long while?

this error will have been caused by a lack of a column in the old sqlite database which was added quite a while back.

**Comment by peppy** [comment, marked as answer]
Please use the attached `client.realm` (put it in your main osu! folder, overwriting the one you have there):
[client.realm.zip](https://github.com/ppy/osu/files/7963793/client.realm.zip)

**Comment by ghost** [comment]
Thank you for replying and the patch file, everything is working well now! 寄件者: Dean Herbert傳送時間: 2022年1月29日 下午 09:45收件者: ppy/osu副本: sky93192; Author主旨: Re: [ppy/osu] Realm migration issue (One or more errors occurred. (SQLite Error 1: 'no such column: s.InstantiationInfo'.)) (Discussion #16700) Please use the attached client.realm (put it in your main osu! folder, overwriting the one you have there):client.realm.zip—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>
